src/app/Caches/Create/AdvancedOptions.tsx

```diff
@@ -0,0 +1,300 @@
+                </TextContent>
+                <TextContent>
+                    <Text component={TextVariants.p}>{t('caches.create.configurations.advanced-options.locking-subtitle')}</Text>
+                </TextContent>
+                <FormGroup
+                    isInline
+                    isRequired
+                    fieldId="field-concurrency-level"
+                >
+                    <MoreInfoTooltip label={t('caches.create.configurations.advanced-options.concurrency-level-title')} toolTip={t('caches.create.configurations.advanced-options.concurrency-level-tooltip')} textComponent={TextVariants.h4} />
+                    <TextInput value={concurrencyLevel} type="number" onChange={handleConcurrencyLevel} aria-label="concurrency-level-input" />
+                </FormGroup>
+                <FormGroup
+                    isInline
+                    isRequired
+                    fieldId="field-isolation-level"
+                >
+                    <MoreInfoTooltip label={t('caches.create.configurations.advanced-options.isolation-level-title')} toolTip={t('caches.create.configurations.advanced-options.isolation-level-tooltip')} textComponent={TextVariants.h4} />
+                    <Select
+                        variant={SelectVariant.single}
+                        aria-label="isolation-level-select"
+                        onToggle={onToggleIsolationLevel}
+                        onSelect={onSelectIsolationLevel}
+                        selections={isolationLevel}
+                        isOpen={isOpenIsolationLevel}
+                        aria-labelledby="toggle-id-isolation-level"
+                    >
+                        {isolationLevelOptions()}
+                    </Select>
+                </FormGroup>
+                <FormGroup
+                    isInline
+                    isRequired
+                    fieldId="field-lock-acquisition-timeout"
+                >
+                    <MoreInfoTooltip label={t('caches.create.configurations.advanced-options.lock-acquisition-timeout-title')} toolTip={t('caches.create.configurations.advanced-options.lock-acquisition-timeout-tooltip')} textComponent={TextVariants.h4} />
+                    <TextInput value={lockAcquisitionTimeout} type="number" onChange={handleLockAcquisitionTimeout} aria-label="lock-acquisition-timeout-input" />
+                </FormGroup>
+                <FormGroup
+                    isInline
+                    fieldId="field-striping"
+                    label="Striping"
+                >
+                    <Checkbox aria-label="striping" id="striping" onChange={onToggleStriping} isChecked={striping} />
+                </FormGroup>
+            </React.Fragment>
+        );
+    }
+
+    return (
+        <React.Fragment>
+            <Form>
+                {formMemory()}
+                <Divider />
+                {formLocking()}
+
+            </Form>
+        </React.Fragment>
```

Comment:
you don't need React.Fragment if you have a begin and close tag. With <\Form\> is enough